docroot/lib/views/home.php

```diff
@@ -19,7 +19,7 @@
             <h1><?php print _("get GNU/Linux!"); ?></h1>
             <div class="slide-text">
               <p><?php # translators: This is the very first sentence every visitor is going to read. Please pay great attention to making it sound great, we only have one try! =)
-              print _("GNU/Linux is an alternative to Microsoft Windows and macOS. It is easy to use and gives more freedom to users. Anyone can install it: GNU/Linux is free as in freedom, and often available free of charge."); ?></p>
+              print _("GNU/Linux is an alternative to Windows, Android, macOS, iOS and Chrome OS. It's easy to use and gives the user total control over data and the computing. Feel free to install it today! GNU/Linux consists of free software&mdash;where free is meant as in freedom but it's often also available free of charge."); ?></p>
```

Comment:
> It's easy to use and gives the user total control over data and the computing.

This sentence feels a bit awkward to me. Maybe change it to something along the lines of:

> It's easy to use and gives the user total control over their computing and their data.